site/pages/code-war.rst

```diff
@@ -0,0 +1,54 @@
+.. title: Friday Warmup Event: Code War
+.. slug: code-war
+.. date: 2018-06-07 23:33:47 UTC+07:00
+.. tags: 
+.. category: 
+.. link: 
+.. description: 
+.. type: text
+
+Eight teams, onstage knockout rounds of short programming bouts, loud crowd,
+mildly impressive prizes. Any language allowed, no holds barred. Think of it
+like cage fighting for coders.
+
+We guarantee this won’t be mind twisting problems for algorithm heads. It will
+help to have a team with a mix of skills, a clever and/or intimidating team
+name and vocal cheer squad.
+
+For example: “Here is the complete text of *Pride and Prejudice* from this URL.
+Be the first to print out the 7th and 25th sentences that contain both
+’Elizabeth’ and ‘Darcy’“.
+
+Come along to watch. Come along to compete. Come along to just hangout and meet
+others. Bring your laptop if you want to try the questions from the audience.
+
+**Pizza is provided!**
+
+Rules of the Game
+=================
+
+1. One laptop per team and no internet use at any time. You will be live coding.
+2. Any programming language you want as long it an your IDE has a large font size. 
```

Comment:
Didn't see this one before, but "as long it an your IDE" seems to be missing bits too.

Reply:
ah!